src/content/developers/docs/scaling/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Scaling
+description: An introduction to the different scaling options currently being developed by the Ethereum community.
+lang: en
+sidebar: true
+incomplete: true
+---
+
+## Scaling Overview {#scaling-overview}
+
+As the number of people using Ethereum has grown, the blockchain in its native form has reached certain capacity limitations. This has driven up the cost of using the network, creating the need for "scaling solutions." There are multiple solutions being researched, tested and implemented that take different approaches to achieving similar goals.
+
+The main goal of scalability is to increase transaction speed (faster finality), and transaction throughput (high transactions per second), without sacrificing decentralization or security (more on [Eth2 vision](/eth2/vision/)). On the layer 1 Ethereum blockchain, high demand leads to slower transactions and nonviable gas prices. Increasing the network capacity in terms of speed and throughput is fundamental to the meaningful and mass adoption of Ethereum. 
+
+While speed and throughput are important, it is essential that scaling solutions enabling these goals remain decentralized and secure. Keeping the barrier to entry low for node operators is critical in preventing a progression towards centralized and insecure computing power.
+
+Conceptually we first categorize scaling as either on-chain scaling or off-chain scaling.
+
+## Prerequisites {#prerequisites}
+
+You should have a good understanding of all the foundational topics. Implementing scaling solutions is advanced as the technology is less battle-tested, and continues to be researched and developed.
+
+## On-Chain Scaling {#on-chain-scaling}
+
+This method of scaling requires changes to the existing layer 1 [mainnet](/glossary/#mainnet). Sharding is currently the main focus for this method of scaling.
+
+### Sharding {#sharding}
```

Comment:
Is sharding the only Eth2 upgrade that improves scaling? Doesn't PoS improve scalability, if anything by providing instant finality on txs/blocks?